scripts/research/b2_01_volatility_transition_lib.py

```
@@ -0,0 +1,1033 @@
+            SEED_PLACEBO,
+            replicate,
+            w_min,
+            d_min,
+            h_min,
+            int(t_ms[i]),
+            stratum_id,
+        )
+        rng = np.random.default_rng(seed)
+        shuffled = labels.copy()
+        rng.shuffle(shuffled)
+        for j, assigned in zip(ordered, shuffled):
```

Ruff (0.16.2)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)